src/main/java/edu/harvard/iq/dataverse/SettingsWrapper.java

```diff
@@ -251,15 +251,6 @@ public boolean isDoiInstallation() {
         }
     }
 
-    public boolean isDataCiteInstallation() {
```

Comment:
There's something weird going on. I expected isDataCiteInstallation to be in use by dataset.xhtml but it's not, in either this branch or the develop branch. It was added in 28f3222 to grey out the publish button if a DOI hasn't been registered. But now this logic is... gone? Do we still want it? Flagging @landreev .

Reply:
I only deleted this method because I couldn't find it being used. If it is, we can restore.

Reply:
@qqmyers you're right that it isn't being used but that's only because something got lost on the xhtml side. Please restore it. For more details, please see #7102.